apps/app/src/views/routes/SettingsV2/apollo.tsx

```diff
@@ -0,0 +1,77 @@
+import {
+  ApolloClient,
+  HttpLink,
+  InMemoryCache,
+  NormalizedCacheObject,
+  useApolloClient
+} from '@apollo/client';
+import { setContext } from '@apollo/client/link/context/index.js';
+import { PhotonClient } from '@photonhealth/sdk';
+import { createContext, useContext } from 'react';
+
+let _apollo: ApolloClient<NormalizedCacheObject> | undefined;
+let _photonClient: PhotonClient | undefined;
+
+const clinicalApiUrl = process.env.REACT_APP_CLINICAL_GRAPHQL_URI as string;
+
+const apollo = (photonClient: PhotonClient) => {
```

Comment:
Note: I specifically want two separate ApolloClients here (at least for now) so that we don't need to deal with conflicting type definitions

Reply:
I think I'm rehashing things we talked about last Friday, but gonna recap some thoughts:

An alternate solution would be to have the SDK have two `HttpLink`s, one for the existing URI and the other for the new one. Then, you could use `ApolloLink.split` to decide which link to use based on some context like:

`useQuery(QUERY, { variables, context: { clientName: 'newClinical' } }) // bad naming`

But, in this solution you're saying that we would have conflicting type definitions because the Apollo client would have collisions on the type names because both endpoints are sharing the same cache. That makes sense ✅

So at least for now, we'll agree this is a necessary evil until we're fully migrated over. That means we'll have two instances of the Apollo client and recognize the hit we're taking (how big? 🤷‍♂️). Probably not something we'd want on the Patient App (tho not even using apollo there)... but fine for now on the clinical app (tho bearing in mind some orgs' clinicians are using their phones).